EIPS/eip-6956.md

```diff
@@ -0,0 +1,515 @@
+---
+eip: 6956
+title: Asset-bound Non-Fungible Tokens
+description: Asset-bound NFTs anchor a token 1-1 to an asset and transfers are authorized through oracle-attestation of control over the asset
+author: Thomas Bergmueller (@tbergmueller) <thomas.bergmueller@authenticvision.com>, Lukas Meyer (@ibex-technology) <lukas@ibex.host>
+discussions-to: https://ethereum-magicians.org/t/erc-6956-asset-bound-non-fungible-tokens/14056
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-29
+requires: 165, 721
+---
+
+## Abstract
+
+This standard allows to integrate physical and digital ASSETS without signing capabilities into dApps/web3 by extending [ERC-721](eip-721.md).
+
+An `ASSET`, e.g. a physical object, is equipped with an `ANCHOR`. The `ANCHOR-TECHNOLOGY` must be chosen s.t. an ANCHOR allows to uniquely identify the ASSET. The ANCHOR-TECHNOLOGY must further allow to establish a `PROOF-OF-CONTROL` over the ASSET through an `ORACLE`. For physical ASSETS, PROOF-OF-CONTROL corresponds for example to proof of physical presence.
```

Comment:
Please expand "s.t." and for all other abbreviations, make sure to expand them the first time they're used.

Reply:
done in https://github.com/ethereum/EIPs/commit/9d188578511cb53c00ebe0910043ab50fb40a248